tests/scripts/tg/run_tg_frequent_tests.sh

```diff
@@ -7,6 +7,8 @@ run_tg_tests() {
   echo "LOG_METAL: running run_tg_frequent_tests"
 
   pytest tests/ttnn/multichip_unit_tests/test_multidevice_TG.py
+  pytest models/demos/tg/llama3_70b/tests/test_llama_mlp_galaxy.py
```

Comment:
i think we should use `pytest -n auto` to use xdist and better timeout in these pipelines

Reply:
To further expand on @vtangTT 's point
It would be good to expand the usage of xdist across all pytests being called for all model tests going forward.
This method lets us catch and handle hanging tests better and better control timeouts. (using xdist would discourage the use of pytest.skip() at the lower level)

Example reference call below.

```
pytest -n auto models/demos/falcon7b/tests/test_perplexity_falcon.py::test_perplexity[True-prefill_seq1024_dram] --timeout=1800 ; fail+=$?
```

Reply:
Modified all pytest calls to use xdist and timeouts

Reply:
> (using xdist would discourage the use of pytest.skip() at the lower level)

@tt-rkim advices that we call `test_llama_mlp_galaxy.py` and use `pytest.skip` to omit tests.

fyi @ttmchiou